src/Util/Frame.php

```diff
@@ -49,7 +60,7 @@ public function getFile()
     }
 
     /**
-     * @return int|null
+     * @return integer|null
```

Comment:
I can hardly say that all the league packages follows this logic:

https://github.com/search?utf8=%E2%9C%93&q=%40return+boolean+user%3Athephpleague+language%3APHP&type=Code&ref=searchresults

https://github.com/search?utf8=%E2%9C%93&q=%40return+integer+user%3Athephpleague+language%3APHP&type=Code&ref=searchresults